cli/lib/kontena_cli.rb

```diff
@@ -87,6 +115,26 @@ def self.cli_root(*joinables)
       File.join(Kontena.root, 'lib/kontena/cli', *joinables)
     end
   end
+
+  def self.logger
+    return @logger if @logger
+    if log_target.respond_to?(:tty?) && log_target.tty?
+      logger = Logger.new(log_target)
+      logger.datetime_format = "%M%S.%2N "
```

Comment:
This datetime format is rather odd, having just minutes and seconds reads funny. Also, the sub-second formatting could have at least three decimals to ms percision?

```
D, [4554.94 #873] DEBUG -- CONFIG: Current master: development
D, [4554.95 #873] DEBUG -- CONFIG: Current grid: development
```